libs/containers/User.lua

```diff
@@ -202,4 +219,18 @@ function get.mutualGuilds(self)
 	return self._mutual_guilds
 end
 
+--[=[@p badges Array An array of all badges the user has, represented by the badge's name.]=]
```

Comment:
This should either say `table` or it should return an `ArrayIterable` instead of a table (I recommend this), `Array` on its own is not a valid type.